### DISCUSSION TITLE
Python Devs required!

### DISCUSSION THREAD

**Discussion by Shreesh-Kulkarni** [NITK-KODE/NITK-KODE#11] [Q&A]
Just wanted to ask if anyone is interested in making open source projects in Python. Could be anything from Data Science related to Computer Vision. Just wanna get started somewhere.

**Comment by danishjuneja** [comment]
Python dev here

**Reply by Shreesh-Kulkarni** [reply]
Cool

**Comment by kid-116** [comment]
How about a trading bot?

**Reply by Shreesh-Kulkarni** [reply]
Using discord.py? I heard that they have stopped the development of that library

**Reply by Shreesh-Kulkarni** [reply]
Also I assumed you meant a discord bot sorry lol.

**Reply by kid-116** [reply]
Yea. Not a discord bot :p

**Reply by Shreesh-Kulkarni** [reply]
Yeah sry, i got this notif on discord thats why lmao . No problem we could discuss more about this
